serializability-checker-compiler-plugin/src/main/resources/scalac-plugin.xml

```diff
@@ -0,0 +1,4 @@
+<plugin>
+    <name>akka-serializability-checker-plugin</name>
+    <classname>org.virtuslab.akkasaferserializer.SerializabilityCheckerCompilerPlugin</classname>
```

Comment:
Okay so rename of the project/packages to ASH will happen on a follow-up PR right?